otehiwai_po/custom_targets.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Thu Nov 16 04:03:40 2023
+
+@author: hopkinsm
+"""
+
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+from astropy.coordinates import SkyCoord
+import astropy.units as u
+from copy import deepcopy
+import os
+from utilly import save_targs, get_today, make_dir, make_obs_entry
+import requests
+from astroquery.mpc import MPC
+
+package_directory = os.path.dirname(os.path.abspath(__file__)) + '/'
```

Comment:
Ideally we should use os.path.join(your_path, '') to provide a trailing slash as then it will work on any system

Reply:
Done and changed all used functions to os.path.join rather than / concatenation.